template/src/themes/metrics.ts

```diff
@@ -94,4 +93,5 @@ export {
   responsiveWidth,
   deviceWidth,
   deviceHeight,
-}
+};
```

Comment:
Remove `;` in this file to match with other files

Reply:
updated